engines/mtropolis/boot.cpp

```diff
@@ -1210,6 +1224,14 @@ const Game games[] = {
 		nullptr,
 		GameDataHandlerFactory<STTGSGameDataHandler>::create
 	},
+	// Unit: Rebooted
+	{
+		MTBOOT_UNIT_REBOOTED_WIN,
+		unitWinFiles,
+		unitWinDirectories,
+		nullptr,
+		GameDataHandlerFactory<STTGSGameDataHandler>::create
```

Comment:
I should probably rename this. I think that the `AlbertGameDataHandler` is basically the same as `STTGSGameDataHandler`.   Maybe it they can be combined with a generic name for the handler that has a `StandardPlugIn`?